server/src/test/java/org/opensearch/indices/IndicesRequestCacheTests.java

```diff
@@ -650,6 +665,186 @@ public long ramBytesUsed() {
         }
 
         @Override
-        public void writeTo(StreamOutput out) throws IOException {}
+        public void writeTo(StreamOutput out) throws IOException {
+        }
+    }
+
+    public static class CleanDiskCacheTests {
```

Comment:
These don't run as part of the test suite in intellij for me - it says bc it's part of the nested class, but it might also be bc of the junit stuff. Should prob refactor this, but I'm gonna go ahead and benchmark without those changes

Reply:
I see that too, let me address this, thanks !